packages/vite/src/node/build.ts

```diff
@@ -133,6 +138,20 @@ export interface BuildOptions {
    * @default minify
    */
   cssMinify?: boolean
+  /**
+   * @experimental
+   * Use Lightning CSS to minify CSS. This should be installed as a peer
+   * dependency. Targets is automatically converted from esbuild (css)target, but
+   * can be provided for more granular control.
+   * @default esbuild
+   */
+  cssMinifier?:
+    | { minifier: 'esbuild' }
+    | {
+        minifier: 'LightningCSS'
+        targets?: LightningCSSTargets
+        drafts?: LightningCSSDrafts
+      }
```

Comment:
We need to discuss in a team meeting what is the best here. Another option would be to have `cssMinify` be a string optionally ( `boolean | 'esbuild' | 'LightingCSS'` ) and then have a new `lightningCssOptions` (aligning with what we have with `minify` and `terserOptions`).

Reply:
We didn't discuss this part at the end of the meeting, WDYT for this one?